### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by webfrontzhifei**
the bug is : when you test the style a arbitrary name，like "margin-top333", the old function will return true, because the value of 'elStyle' and 'mockNodeStyle' are all ''undefined"

### CODE REVIEW

**Comment by webfrontzhifei** [comment]
it('Message component has style margin-top2: 10', () => {
     expect(cmp.find(Message).hasStyle('margin-top2', '10')).toBe(true)
   })

   The test Result is :
   ✓ Message component has style margin-top2: 10 (35ms)

**Comment by eddyerburgh** [comment]
Can you add that to the tests/unit/specs/Wrapper/hasStyle.spec.js in this PR

**Comment by webfrontzhifei** [comment]
ok, i have added it.

File: src/wrappers/wrapper.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ export default class Wrapper implements BaseWrapper {
 
     const elStyle = window.getComputedStyle(this.element)[style]
     const mockNodeStyle = window.getComputedStyle(mockNode)[style]
-    return elStyle === mockNodeStyle
+    return elStyle && mockNodeStyle && elStyle === mockNodeStyle
```
Comment (eddyerburgh) [inline src/wrappers/wrapper.js]: You need to wrap this whole return statement in parentheses and cast them to a Boolean:
```
return !!(elStyle && mockNodeStyle && elStyle === mockNodeStyle)
```

Comment (webfrontzhifei) [reply src/wrappers/wrapper.js]: ok, i have fixed it

**Pull request merged**